tests/js/index.js

```diff
@@ -28,8 +28,8 @@ var TESTS = {
     QueryTests: require('./query-tests'),
     EncryptionTests: require('./encryption-tests'),
     MigrationTests: require('./migration-tests'),
-    UserTests: require('./user-tests'),
-    AsyncTests: require('./async-tests')
+    UserTests: require('./user-tests')
+    /*AsyncTests: require('./async-tests')*/
```

Comment:
why are we disabling these? isn't this the only place we actually test collection notifications?